cmd/tke-installer/app/installer/installer.go

```diff
@@ -411,7 +413,7 @@ func (t *TKE) initSteps() {
 	if !t.Para.Config.Registry.UseDevRegistry() {
 		t.steps = append(t.steps, []handler{
 			{
-				Name: fmt.Sprintf("Push images to %s/%s", t.Para.Config.Registry.Domain(), t.Para.Config.Registry.Namespace()),
```

Comment:
Why remove registry info?

Reply:
Because a static step name is more friendly when it's used in `skipSteps`.